### PR TITLE
Various Website Fixes

### LINKED ISSUE #1095: Page titles for docs page (remove the redundant prefixes)
Example below...see the left-hand nav when you open https://trufflesuite.com/docs/drizzle

Screen Shot 2021-12-03 at 12 42 05 PM

### LINKED ISSUE #1098: Hovering over the search input makes the "Search" placeholder text invisible (while in light mode)

### LINKED ISSUE #1109: Markdown Not Rendering Correctly on Tutorial
The [Expand This Section](http://trufflesuite.com/tutorial/#testing-the-smart-contract-using-javascript) markdown isn't correctly rendering. It shows back ticks and asterisks rather than in-line code and bullets.

image

### PR DESCRIPTION
**Pull request by MicaiahReid**
- Fixes #1098 by changing the placeholder text color of the Search bar in `extra.css`.
 - Fixes #1109 by adding python's md_in_html markdown extension and using it on markdown within `<details>` tags.
 - Fixes #1095 by renaming Overview page for drizzle/ganache/truffle.
 - Fixes logo image and badge on overview pages.

### CODE REVIEW

File: src/docs/drizzle/.pages [hunk 1]
```diff
@@ -0,0 +1,6 @@
+nav:
+  - Overview: index.md
+  - quickstart.md
+  - getting-started
+  - react
+  - reference
```
Comment (lsqproduction) [inline src/docs/drizzle/.pages:6]: Needing a new line at the end of the file, same for below :-)

Comment (MicaiahReid) [inline src/docs/drizzle/.pages:6]:
```suggestion
  - reference
  
```

File: src/docs/ganache/.pages [hunk 2]
```diff
@@ -0,0 +1,7 @@
+nav:
+  - Overview: index.md
+  - quickstart.md
+  - corda
+  - reference
+  - truffle-projects
+  - workspaces
```
Comment (MicaiahReid) [inline src/docs/ganache/.pages:7]:
```suggestion
  - workspaces
  
```

File: src/docs/drizzle/.pages (hunk 1 above)
Comment (MicaiahReid) [reply src/docs/drizzle/.pages]: Fixed!

**Review by lsqproduction — APPROVED** [review]
Looks happy now 💯

**Pull request merged**